gemelli/preprocessing.py

```diff
@@ -94,12 +97,13 @@ def rclr(T):
         M = T.reshape(np.product(T.shape[:len(T.shape) - 1]),
                       T.shape[-1])
         with np.errstate(divide='ignore', invalid='ignore'):
-            M_rclr = rclr_matrix(M)
+            M_tensor_rclr = rclr(M)
+        M_tensor_rclr[~np.isfinite(M_tensor_rclr)] = 0.0
         # reshape to former tensor and return tensors
-        return M_rclr.reshape(T.shape).transpose(reverse_T)
+        return M_tensor_rclr.reshape(T.shape).transpose(reverse_T)
 
 
-def rclr_matrix(M):
+def rclr(M):
```

Comment:
Here it is. I would strongly caution avoiding this name change. `matrix` may make sense as the default. BUT switching the default will strongly affect any user code. Would probably suggest keeping `rclr_matrix`, and `rclr_tensor`, and keeping `rclr` as `tensor`. Possible to deprecate, but I would not just go around switching the API.

Reply:
these have been changed to matrix_rclr and tensor_rclr